ChromeExtension/groups.html

```diff
@@ -15,47 +16,70 @@
     <script type="text/javascript" src="js/knockout.js"></script>
+    <script type="text/javascript" src="js/background.js"></script>
     <script type="text/javascript" src="js/groups.js" defer="defer"></script>
 
 </head>
 
 <body id="body">
-    <div name="contentwrap" style="padding: 15px">
+    <div name="contentwrap" style="padding: 20px">
         <center>
-            <img src="images/logo.svg" style="margin-bottom: 30px" alt="Geniuslink: The world's most intelligent link shortener for marketing products across a fragmented audience." height="80" width="300">
+            <button class="btn-gl-blue" style="width: 250px; font-size: 13px; " data-bind="event: { click: createLinkFromButton }">New link from current page</button>
+            <p class="fine-print" style="position: relative; top: 8px;">Or, right-click a link and select "Create..."</p>
+        </center>
+        <hr class="hr-group">
+        <div class="menu-group">
+
+            <a href="apikeys.html" class="menu-item text-left">Change API Keys
+                        <span class="glyphicon glyphicon-chevron-right pull-right menu-icons" aria-hidden="true"></span>
+                         </a>
+
+
+
+
+            <a href="lastLinks.html" class="menu-item text-left">View last five links
+                        <span class="glyphicon glyphicon-chevron-right pull-right menu-icons" aria-hidden="true"></span>
+                         </a>
+
+
+            <a href="customDomain.html" class="menu-item text-left">Select a custom domain
+                        <span class="glyphicon glyphicon-chevron-right pull-right menu-icons" aria-hidden="true"></span>
+                          </a>
+
+            <a href="help.html" class="menu-item text-left">How to use this plugin
+                        <span class="glyphicon glyphicon-chevron-right pull-right menu-icons" aria-hidden="true"></span>
+                          </a>
+
+
+
+
+        </div>
+        <hr class="hr-group">
+        <div class="group-div">
             <div class="text-left">
-                <label>Put new links in this group</label>
+                <p class="newlink-group">Put new links in this group</p>
                 <select id="listOfGroups" class="form-control">
                     <option id="loadingOption" value="Loading">Loading...</option>
                 </select>
-
-
             </div>
-            <p id="result">&nbsp;</p>
-            <div>
-                <a href="apikeys.html">
-                    <p class="text-left" style="font-size: 12px;">Change API Keys</a></p>
-
-                        <a href="lastLinks.html">
-                            <p class="text-left" style="font-size: 12px;">View last five links</a></p>
-
-                                <a href="help.html">
-                                    <p class="text-left" style="font-size: 12px; ">How to use this plugin</a></p>
 
+            <center>
+                <p style="margin-bottom: -20px;" id="result">&nbsp;</p>
```

Comment:
Oh, looks like you are using it to override the container padding like we did with the HR. That's fine.